packages/web3-wallet/webpack.config.cjs

```diff
@@ -41,12 +41,15 @@ module.exports = {
     }
   },
   output: {
-    filename: 'alephium-web3-wallet.min.js',
+    filename: '_umd/alephium-web3-wallet.min.js',
     library: {
       name: 'alephium',
       type: 'umd'
     }
   },
+  experiments: {
+    asyncWebAssembly: true
```

Comment:
Question: why do we need to add this to the `web3-wallet` package?

Reply:
I am not sure if this is still necessary, but I copied it from `web3`. IIRC, we needed it in the past for some project.